src/cinder/Log.cpp

```diff
@@ -502,6 +521,64 @@ void LoggerSysLog::write( const Metadata &meta, const string &text )
 	syslog( sysLevel , "%s %s", meta.toString().c_str(), text.c_str() );
 }
 
+#elif defined( CINDER_MSW )
+
+// ----------------------------------------------------------------------------------------------------
+// MARK: - LoggerEventLog
+// ----------------------------------------------------------------------------------------------------
+
+LoggerEventLog::LoggerEventLog()
+{
+	char filename[MAX_PATH];
+	wchar_t wFilename[MAX_PATH];
+	string stem;
+
+	DWORD size = GetModuleFileNameA( NULL, filename, MAX_PATH );
```

Comment:
We haven't documented this much, but when calling platform specific C API's, especially for windows or mac, we prepend a `::` to ensure the name comes from the global namespace, also documenting that fact.